gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,10 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.L1A_ID")
```

Comment:
These are not probably what you want.
* Do you want a count of the number of triggers seen at the AMC?
  * `GEM_AMC.TTC.COUNTERS.L1A`
* Do you want the number of triggers seen since the DAQ module was reset?
  * Then what you have will do this, but not all running modes will enable the DAQ link and event building, so this is probably the wrong way to count.